cmd/list/firmware.go

```diff
@@ -45,3 +76,13 @@ var listFirmware = &cobra.Command{
 		table.Render()
 	},
 }
+
+func init() {
+	flagsDefinedListFirmware = &listFirmwareFlags{}
+
+	listFirmware.PersistentFlags().StringVar(&flagsDefinedListFirmware.server, "server", "", "server UUID")
+	listFirmware.PersistentFlags().StringVar(&flagsDefinedListFirmware.vendor, "vendor", "", "vendor name")
+	listFirmware.PersistentFlags().StringSliceVar(&flagsDefinedListFirmware.model, "model", make([]string, 0), "list of models separated by commas")
```

Comment:
It's not a huge thing, but `nil` is a valid slice value (it's the 0-value), and it's valid to be appended-to, ranged-over, or otherwise manipulated.

Reply:
oh ok, i didnt know that. Thanks, i change that.